### PR TITLE
Run tests on 32 bit images

### PR DESCRIPTION
**Pull request by MarcelGarus**

### CODE REVIEW

**Review by gritfessel — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 name: CI
 
 # Controls when the action will run. Triggers the workflow on push or pull request events on every branch
-on: [ push, pull_request ]
+on: [push, pull_request]
```
Comment (SilvanVerhoeven) [inline .github/workflows/ci.yml:6]: Just to be sure: So we don't want extra spaces on the inside of brackets?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -12,9 +12,29 @@ jobs:
     strategy:
       matrix:
         # Select platform(s)
-        os: [ ubuntu-latest, macos-latest, windows-latest ]
+        os: [ubuntu-latest, macos-latest, windows-latest]
```
Comment (SilvanVerhoeven) [inline .github/workflows/ci.yml:15]: see comment above

**Review by SilvanVerhoeven — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (marcelgarus) [reply .github/workflows/ci.yml:6]: I think both options are okay, but that's what the VSCode default yaml formatter suggested…

**Comment by marcelgarus** [comment]
For our linting, we depend on SWALint and SWALint apparently doesn't work on 32bit images.
So we're leaving the current situation as-is for now.

**Pull request closed without merging**